src/composables/transactionList.ts

```diff
@@ -1,30 +1,33 @@
-import { orderBy, uniqBy } from 'lodash-es';
-import { ref, watch } from 'vue';
-import { Encoded, Tag } from '@aeternity/aepp-sdk';
+import { computed, ref, watch } from 'vue';
+import { Encoded } from '@aeternity/aepp-sdk';
 import type {
   ITransaction,
   ITransactionsState,
   IAccountTransactionsState,
   IDefaultComposableOptions,
+  Protocol,
 } from '@/types';
 import {
   TRANSACTIONS_LOCAL_STORAGE_KEY,
   TX_DIRECTION,
+  TXS_PER_PAGE,
 } from '@/constants';
 import {
-  fetchJson,
   getLocalStorageItem,
   setLocalStorageItem,
 } from '@/utils';
-import JsonBig from '@/lib/json-big';
-import { AEX9_TRANSFER_EVENT } from '@/protocols/aeternity/config';
-import { useAeNetworkSettings } from '@/protocols/aeternity/composables';
 
+import { ProtocolAdapterFactory } from '@/lib/ProtocolAdapterFactory';
+import { AE_MDW_TO_NODE_APPROX_DELAY_TIME, AEX9_TRANSFER_EVENT } from '@/protocols/aeternity/config';
+import { useMiddleware } from '@/composables/middleware';
```

Comment:
I don;t understand why you changed the paths here. This is local import from the same directory. Please revert this change.

Reply:
this is related to problem with load adapter. You found solution and you wrote to me how import useMiddleware

Reply:
It's not about splitting the composables into separate imports as this was done this way before. My component referst to the fact that you changed the import paths for files that are imported from the same directory. Before it was `./middleware` and you changed it to `@/composables/middleware` which makes no sense here.